xcell/mappers/mapper_base_Planck.py

```diff
@@ -0,0 +1,85 @@
+import healpy as hp
+import pymaster as nmt
+from .mapper_base import MapperBase
+
+
+class MapperBasePlanck(MapperBase):
+    def __init__(self, config):
+        self._get_Planck_defaults(config)
+        return
+
+    def _get_Planck_defaults(self, config):
+        self._get_defaults(config)
+        self.file_map = config['file_map']
+        self.file_hm1 = config.get('file_hm1', None)
+        self.file_hm2 = config.get('file_hm2', None)
+        self.file_mask = config.get('file_mask', None)
+        self.file_gp_mask = config.get('file_gp_mask', None)
+        self.file_sp_mask = config.get('file_sp_mask', None)
+        self.signal_map = None
+        self.hm1_map = None
+        self.hm2_map = None
+        self.diff_map = None
+        self.nl_coupled = None
+        self.cl_coupled = None
+        self.cl_mode = config.get('cl_mode', 'Auto')
+        self.cls = {'Auto': None,
+                    'Cross': None}
+        self.custom_auto = True
```

Comment:
Reminder for myself: are you using this anywhere?

Reply:
Each instance of the class can only fill one at the moment so we can remove it if we don't want to make use of it

Reply:
After chatting we agreed on:
- We remove the `cl_mode` and `Auto/Cross` options.
- Keep the sef.cls_coupled as an attribute, since that's handy.
- Make `custom_auto` an attribute of all mappers by adding it to the base class with a default value of `False`

Reply:
Done